index.html

```diff
@@ -0,0 +1,780 @@
+            <p class="dennis">
+                By Dennis Overbye
+            </p>
+        </div>
+        <div class="dennis-logo">
+            <p class="dennis-date">March 17, 2014</p>
+            <ion-icon name="mail-outline" class="mainOutline"></ion-icon>
+            <ion-icon name="arrow-redo-circle-outline" class="redoCircle"></ion-icon>
+            <ion-icon name="bookmark-outline" class="bookmark"></ion-icon>
+            <ion-icon name="chatbox-outline" class="chatbox"></ion-icon>
+        </div>
+    </div>
+
+    <div class="cambridgeMass">
+        <p>
+            CAMBRIDGE, Mass. — One night late in 1979, an itinerant young physicist named Alan Guth,
+            with a new son and a year’s appointment at Stanford, stayed up late with his notebook and equations,
+            venturing far beyond the world of known physics.
+        </p>
+        <p>
+            He was trying to understand why there was no trace of some exotic particles that should have been created in
+            the Big Bang.
+            Instead he discovered what might have made the universe bang to begin with.
+            A potential hitch in the presumed course of cosmic evolution could have infused space itself with a special
+            energy that exerted a repulsive force,
+            causing the universe to swell faster than the speed of light for a prodigiously violent instant.
+        </p>
+        <p>
+            If true, the rapid engorgement would solve paradoxes like why the heavens look uniform from pole to pole
+            and not like a jagged, warped mess. The enormous ballooning would iron out all the wrinkles and
+            irregularities.
+            Those particles were not missing, but would be diluted beyond detection, like spit in the ocean.
+        </p>
+        <p>
+            “SPECTACULAR REALIZATION,” Dr. Guth wrote across the top of the page and drew a double box around it.
+        </p>
+        <p>
+            On Monday, Dr. Guth’s starship came in. Radio astronomers reported that they had seen the beginning of the
+            Big Bang, and that his hypothesis, known undramatically as inflation, looked right.
+        </p>
+        <p>
+            Reaching back across 13.8 billion years to <a href="">first sliver of cosmic time</a> with telescopes at the
+            South
+            Pole, a team of astronomers led by John M. Kovac of the Harvard-Smithsonian Center for Astrophysics detected
+            ripples in the fabric of space-time — so-called <a href="#">gravitational waves</a> — the signature of a
+            universe being
+            wrenched violently apart when it was roughly a trillionth of a trillionth of a trillionth of a second old.
+            They are the long-sought smoking-gun evidence of inflation, proof, Dr. Kovac and his colleagues say, that
+            Dr. Guth was correct.
+        </p>
+
+        <!----------------Thanks for reading the Times section------------------->
```

Comment:
Remove this comment because it doesn't highlight anything about your times section